### PR TITLE
deployment: update documentation.

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

**Comment by pramodk** [comment]
@alkino : when time permits, could you take a look at documentation? thanks!

**Comment by matz-e** [comment]
Ping @alkino

**Comment by matz-e** [comment]
Feedback on the [redeplyment description](https://github.com/BlueBrain/spack/blob/deploy/docs/deploy/Workflows.md) would be appreciated.

File: deploy/Concepts.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-# SpackD: Deploying Software Stacks with Spack
+# Conceptual Deployment of Software Stacks with Spack
 
-This is work-in-progress instructions for deploying software stack with
-Spack.
+These instructions were compiled in prepariation for our continuous
```
Comment (alkino) [inline deploy/Concepts.md]: preparation

File: deploy/Concepts.md [hunk 2]
```diff
@@ -18,14 +20,14 @@ Spack.
...
-cd spack-deploy
+git clone https://github.com/BlueBrain/spack.git
+cd spack/deploy
 ```
 
 Setup virtual environment:
 
 ```
-DEPLOYMENT_VIRTUALENV=`pwd`/spackd-venv
+DEPLOYMENT_VIRTUALENV=${PWD}/venv
```
Comment (alkino) [inline deploy/Concepts.md]: always put " around variables

**Review by alkino — DISMISSED** [review]
Seems good to me

**Comment by pramodk** [comment]
@matz-e : after applying Nico's suggestions, should we press merge button?

**Comment by matz-e** [comment]
Yes

**Comment by matz-e** [comment]
@alkino , @pramodk  re-review?

**Review by pramodk — APPROVED** [review]

**Review by alkino — APPROVED** [review]

**Pull request merged**